src/GitReleaseManager.Cli/GitReleaseManager.Cli.csproj

```diff
@@ -1,16 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
-        <LangVersion>8.0</LangVersion>
         <OutputType>Exe</OutputType>
         <AssemblyName>GitReleaseManager</AssemblyName>
-        <TargetFrameworks>net6.0</TargetFrameworks>
         <PublishSingleFile>true</PublishSingleFile>
         <SelfContained>true</SelfContained>
         <RuntimeIdentifier>win-x64</RuntimeIdentifier>
         <Title>GitReleaseManager.Cli</Title>
```

Comment:
Depending on the answer to one of the above comments, shouldn't this one be removed?